### ISSUE TITLE
Radio markType not working

### ISSUE THREAD

**Issue by alexisbronchart** [Marxon13/M13Checkbox#97]
I'm creating the checkbox programmatically like this (in a collectionview cell).
However the component still shows a checkmark and not a radio button style.
Am I missing something?

```
class SubcategoryCell: UICollectionViewCell {
    lazy var checkBox = M13Checkbox(frame: .zero)
    lazy var textLabel = UILabel()
    
    override init(frame: CGRect) {
        super.init(frame: frame)
        
        self.checkBox.tintColor = .qtmPurple
        self.checkBox.secondaryTintColor = .qtmGold
        self.checkBox.secondaryCheckmarkTintColor = .qtmGold
        self.checkBox.stateChangeAnimation = .bounce(.fill)
        self.checkBox.markType = .radio
        self.checkBox.boxType = .square
        
        self.contentView.addSubview(self.checkBox)
        self.contentView.addSubview(self.textLabel)
        
        self.checkBox.snp.makeConstraints { (maker) in
            maker.left.equalToSuperview().offset(16)
            maker.centerY.equalToSuperview()
            maker.height.equalTo(16)
            maker.width.equalTo(16)
        }
        
        self.textLabel.snp.makeConstraints { (maker) in
            maker.top.equalToSuperview().offset(8)
            maker.left.equalTo(checkBox.snp.right).offset(8)
            maker.right.equalToSuperview().offset(-8)
            maker.bottom.equalToSuperview().offset(-8)
        }
    }
    
    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
}
```

**Comment by andreaantonioni** [comment]
@alexisbronchart there's a problem in the code with the property `markType`.
Just use `self.checkbox.setMarkType(type: .radio, animated: false)` instead and it will work 👍🏻

I've fixed the bug in my pull request #98. Hope it will be merged soon!

**Comment by alexisbronchart** [comment]
Great! I'll check if your fix works! (The workaround does)
